### DISCUSSION TITLE
Releases for individual components

### DISCUSSION THREAD

**Discussion by micahhausler** [tinkerbell/tink#554] [General]
This was discussed in the [Tinkerbell community meeting on 11/9/2021](https://docs.google.com/document/d/1cEObfvQ9Tdp8zIIIg9O7P5i3CKaSj2t3JTxEufDxwWs/edit?_hsmi=95274993&_hsenc=p2ANqtz-_hOk14E6kqohN-mM9-b6FwJ5ZG98yoT-iydtVrQEjMAEyJaGDzY4Buu5zekT1yE-rdUn7yVTvkS6vJyCv5oyVAu6GZlA#heading=h.m99826wayf6k), but the individual repos for Tinkerbell, Boots, Hegel, etc do not have semantic, tagged versions/releases. The [sandbox](https://github.com/tinkerbell/sandbox) has tagged releases, but as we incorporate larger changes like the [Kubernetes Resource Model](https://github.com/tinkerbell/proposals/pull/46), we will require tagged releases for API versions and feature changes. 

I wanted to start a discussion here about how we tag and release versions for various components. Does anyone have thoughts they'd like to share?

**Comment by displague** [comment]
We discussed this a bit in the community meetings and some triage parties.
The end result was that we now have semver releases on tink, hegel, and boots matching the sandbox semver tags. The independent projects were tagged according to the sha that sandbox was dependent on.

https://github.com/tinkerbell/tink/releases
https://github.com/tinkerbell/hegel/releases
https://github.com/tinkerbell/boots/releases

A next step will be to ensure that we can create new semver tags on these projects and have that propagate into the images.  The independent projects do not need to _follow_ sandbox versions from this point on.
